woocommerce/woocommerce-android
# Update ProductAddon with latest FluxC model parsing

Summary
==========
This PR fixes a compatibility issue between FluxC develop branch head and our Woo develop branch head, this needs to be merged before the release process is started by the end of the sprint or another PR the current FluxC version here.

How to Test
==========
just make sure tests are green and the code compiles just fine, no changed behavior.


Update release notes:
